docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -169,218 +168,267 @@ Example
-------------------------
+---------------------------------
+Stage 2. Dimensionality reduction
+---------------------------------
 
-The first step in the fitting procedure is to apply a dimensionality reduction to the block matrix *X* using the
-``RidgeReducer``.
+Having the block genotype matrix, the first stage is to apply a dimensionality reduction to the block matrix :math:`X` using the ``RidgeReducer``. After ``RidgeReducer`` is initialized, dimensionality reduction is accomplished within two steps:
 
-This is accomplished by fitting multiple ridge models within each block *x* and producing a new block matrix where each
-column represents the prediction of one ridge model applied within one block. This approach to model building is
-generally referred to as **stacking**. We will call the block genotype matrix we started with the **level 0** matrix in
-the stack *X0*, and the output of the ridge reduction step the **level 1** matrix *X1*. The ``RidgeReducer`` class is
-used for this step, which is initialized with a list of ridge regularization values (referred to here as alpha). Since
-ridge models are indexed by these alpha values, the ``RidgeReducer`` will generate one ridge model per value of alpha
-provided, which in turn will produce one column per block in *X0*, so the final dimensions of matrix *X1* will be
-*Nx(LxK)*, where *L* is the number of header blocks in *X0* and *K* is the number of alpha values provided to the
-``RidgeReducer``. In practice, we can estimate a span of alpha values in a reasonable order of magnitude based on
-guesses at the heritability of the phenotype we are fitting.
+1. Model fitting, performed by the ``RidgeReducer.fit`` function, which fits multiple ridge models within each block :math:`x`.
+2. Model transformation, performed by the  ``RidgeReducer.transform`` function, which produces a new block matrix where each column represents the prediction of one ridge model applied within one block.
 
-Initialization
-==============
+This approach to model building is generally referred to as **stacking**. We call the starting block genotype matrix the **level 0** matrix in the stack, denoted by :math:`X^0`, and the output of the ridge reduction step the **level 1** matrix, denoted by :math:`X^1`. The ``RidgeReducer`` class is initialized with a list of ridge regularization values (here referred to as alpha). Since ridge models are indexed by these alpha values, the ``RidgeReducer`` will generate one ridge model per value of alpha provided, which in turn will produce one column per block in :math:`X^0`. Therefore, the final dimensions of :math:`X^1` for a single phenotype will be :math:`N \times (L \times K)`, where :math:`L` is the number of header blocks in :math:`X^0` and :math:`K` is the number of alpha values provided to the ``RidgeReducer``. In practice, we can estimate a span of alpha values in a reasonable order of magnitude based on guesses at the heritability of the phenotype we are fitting.
```

Comment:
Is the superscript standard notation? I would expect subscripts

Reply:
Wanted to avoid confusion with element index. But not so much important. Changed to subscript.